cardano-cli/src/Cardano/CLI/Run/Friendly.hs

```diff
@@ -315,7 +315,7 @@ friendlyCertificate =
       StakeAddressDeregistrationCertificate credential ->
         "stake address deregistration"
           .= object [friendlyStakeCredential credential]
-      StakeAddressDelegationCertificate credential poolId ->
+      StakeAddressPoolDelegationCertificate credential poolId ->
```

Comment:
Rename `StakeAddressDelegationCertificate` to `StakeAddressPoolDelegationCertificate` so that we can introduce `StakeAddressDRepDelegationCertificate` later.